builtin/mainmenu/init.lua

```diff
@@ -24,6 +24,11 @@ mt_color_dark_green = "#25C191"
 
 local menupath = core.get_mainmenu_path()
 local basepath = core.get_builtin_path()
+local menustyle = core.settings:get("main_menu_style")
+local menustyle = menustylesetting
+if menustyle == "default" then
```

Comment:
```lua
local menustyle = core.settings:get("main_menu_style") or "default"
if menustyle == "default" then
	menustyle = PLATFORM == "Android" and "simple" or "full"
end
```

Reply:
Oops, I could swear I had changed this in my last commit. Will do immediately. 
Also, what does the `or "default"` do? In settingtypes.txt, I set the default to the value `default`, so I don't think that should be needed. But this is the only lua I've ever written, so I don't know.

Reply:
`core.settings:get` does not care about `settingtypes.txt`. That file is only used (yet) to display accurate information in the Advanced Settings dialog.

Reply:
Oh ok. Where does it check? I've been looking at the code, but I'm having trouble figuring it out. Do I just need to add it to defaultsettings.cpp?
Also, in `minetest/src/script/lua_api/l_mainmenu.cpp`, I see `	lua_pushstring(L,"display_width");` and `lua_pushstring(L,"display_height");`. Are these available from lua then, because I've been looking for a way to check screen size rather than platform, for tablets.

Reply:
Removed menustylesetting reference, added to defaultsettings.cpp